### PR TITLE
fix: Solve PaddingMask.Top with StatusBarTranslucent

### LINKED ISSUE #8740: [VisibleBoundPadding] Padding from visibleBoundPadding changes when you rotate the screen with windowtranslucentStatus enabled
### Current behavior

androidwithbug
 The padding from visiblebound padding changes when you rotate the screen to landscape and back to portrait,

### Expected behavior

iosnotbug
Just like on IOS, or on android when android:windowTranslucentStatus is set to false, the padding should not change when you put the phone to portrait mode and back.

### How to reproduce it (as minimally and precisely as possible)

Open the app and rotate the screen to landscape mode. 
Put it back in portait mode.
[landscapeandroid.zip](https://github.com/unoplatform/uno/files/8664120/landscapeandroid.zip)
 

### Workaround

_No response_

### Works on UWP/WinUI

_No response_
The screen needs to rotate to see the bug, so, I didnt try.

### Environment

Uno.UI / Uno.UI.WebAssembly / Uno.UI.Skia

### NuGet package version(s)

image
image

The sample I gave currently has the latest stable Uno.Ui, but the behavior is exactly the same.

### Affected platforms

Android

### IDE

Visual Studio 2022

### IDE version

17.0.6

### Relevant plugins

_No response_

### Anything else we need to know?

image
If you close the screen (or put the phone to sleep? I'm not how to say it well) the padding moves in a different way as you can see in this screenshot, where the rectangle is at the right place, but the background of the grid which is supposed to surround the rectangle, is not.  

I produce this bug on a Samsung Galaxy A70 Android version 11

### PR DESCRIPTION
**Pull request by rafaelrmoukc**
GitHub Issue (If applicable): closes #6216 closes #6218 closes #8740

## PR Type

What kind of change does this PR introduce?

- Bugfix

## What is the current behavior?

When set Android StatusBar color at Style the IsStatusBarTranslucent return false and not set right TOP and X values for newVisibleBounds, with that all Elements are behind of Android StatusBar

## What is the new behavior?

When set Android StatusBar color at Style all Elements are above the Android StatusBar

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [X] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [X] Validated PR `Screenshots Compare Test Run` results.
- [X] Contains **NO** breaking changes
- [X] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [X] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

### CODE REVIEW

**Comment by jeromelaban** [comment]
Make sure to rebase and squash your PR. You also have https://github.com/unoplatform/uno/pull/9033 as a duplicate. Please keep this PR opened and use git force push to update it as needed.

**Comment by rafaelrmoukc** [comment]
Solve #8740 too

File: src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/PopupTests/UnoSamples_Popup_HVAlign_Tests.cs [hunk 1]
```diff
@@ -42,9 +42,13 @@ public void Popup_PlacementTest(string targetName, float xMul, float yMul)
 			var expectedX = buttonRect.X + (buttonRect.Width * xMul);
 			var expectedY = buttonRect.Y + (buttonRect.Height * yMul);
 
+#if __ANDROID__
+expectedY -= 24;
```
[the remark this answers is not in the document]
Comment (rafaelrmoukc) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/PopupTests/UnoSamples_Popup_HVAlign_Tests.cs:46]: I Will not use that, is only a test. I Will got the TOP Padding to make that calc.

[the remark this answers is not in the document]
Comment (rafaelrmoukc) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/PopupTests/UnoSamples_Popup_HVAlign_Tests.cs:46]: Any way to get TOP Padding on UITest without need to set a TextBlock at xaml?

[the remark this answers is not in the document]
Comment (jeromelaban) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/PopupTests/UnoSamples_Popup_HVAlign_Tests.cs]: Do you mean that you want to get the VisibleBounds of the Window? Not at this time. You can write it to a custom DependencyProperty as well, but you'll still need to parse it on the other side.

[the remark this answers is not in the document]
Comment (rafaelrmoukc) [reply src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/PopupTests/UnoSamples_Popup_HVAlign_Tests.cs]: I Will do that as a Dependency, maybe anyone can use that on future, ya?

**Comment by jeromelaban** [comment]
Some tests are failing, here's one:
image

**Comment by rafaelrmoukc** [comment]
> Some tests are failing, here's one: image

Yes, i work at that.

**Comment by rafaelrmoukc** [comment]
@jeromelaban after add "if activity.Window.Attributes.Flags.HasFlag(WindowManagerFlags.DrawsSystemBarBackgrounds)", the UI Test start to insert a "status bar" before "navigation bar", see the screenshot before and after add that.

Before:
Screenshot_1656036408

After:
Screenshot_1656037411

What u think I can do about?

@iury-kc

**Comment by iury-kc** [comment]
Let's check it together. Will be easier. I didn't look at this part in UNO yet, so i may talking nonsense, however, do we have something like android fitsSystemWindows implemented on UNO?

**Comment by rafaelrmoukc** [comment]
@jeromelaban can u see here?

**Comment by jeromelaban** [comment]
It's now preferred to use `SafeArea`.

**Pull request closed without merging**